stl/inc/compare

```diff
@@ -12,7 +12,11 @@
 #if !_HAS_CXX20
 #pragma message("The contents of <compare> are available only with C++20 or later.")
 #else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
+#ifdef __cpp_lib_concepts

```

Comment:
can we mark this with a transition and a GH bugnumber, so we remember to remove this conditional stuff once all our front-ends have __cpp_lib_concepts?

Reply:
Hmmm I don't think we have precedent for that; we generally only do bugs for features that are claimed to be implemented but are broken. (And a GH item is really only appropriate if it's our bug rather than a bug in a frontend)